Explorer/Assets/DCL/PluginSystem/World/SceneUIPlugin.cs

```diff
@@ -70,8 +71,9 @@ public void InjectToWorld(ref ArchSystemsWorldBuilder<Arch.Core.World> builder,
             UIInputReleaseSystem.InjectToWorld(ref builder, componentPoolsRegistry);
             UIDropdownInstantiationSystem.InjectToWorld(ref builder, componentPoolsRegistry);
             UIDropdownReleaseSystem.InjectToWorld(ref builder, componentPoolsRegistry);
+            UIPointerEventsSystem.InjectToWorld(ref builder, sharedDependencies.SceneStateProvider, sharedDependencies.EcsToCRDTWriter);
 
-            finalizeWorldSystems.Add(ReleasePoolableComponentSystem<VisualElement, UITransformComponent>.InjectToWorld(ref builder, componentPoolsRegistry));
+            finalizeWorldSystems.Add(ReleasePoolableComponentSystem<UITransformComponent, UITransformComponent>.InjectToWorld(ref builder, componentPoolsRegistry));
```

Comment:
Thus this line is not needed either

Reply:
Done here: https://github.com/decentraland/unity-explorer/pull/290/commits/9a2080718bba560e36d06fe903ad7986ee6fa5db